tests/storage/export.go

```diff
@@ -30,6 +30,8 @@ import (
 
```

Comment:
I don't see `Skipped` phase tested anywhere?

Reply:
I added an extensive unit test for skipped. And decided to not add a functional test. If you insist I can add one.